### ISSUE TITLE
Images for testing purposes

### ISSUE THREAD

**Issue by gschulze** [georust/geotiff#20]
I'm looking for GeoTIFFs for testing purposes. In particular, I want something like https://download.osgeo.org/geotiff/samples/GeogToWGS84GeoKey/GeogToWGS84GeoKey5.tif for testing various coordinate transformation methods, with easily identifiable control points:

1. One image using the ModelTiePoint and PixelScale tags
2. One image using the ModelTransformation tag
3. One image using multiple ModelTiePoints (or ground control points)

I'm currently trying to prepare such images, but maybe I can save some work by using existing images. Do you have an idea where to look, @weiji14, @dominikbucher, or @kylebarron?

**Comment by kylebarron** [comment]
I don't know but @geospatial-jeff and @vincentsarago might

**Comment by kylebarron** [comment]
Check out https://github.com/OSGeo/gdal/tree/master/autotest/gcore/data

**Comment by weiji14** [comment]
Yeah, I wish there was a standard set of GeoTIFF files for testing somewhere. Generally I try and find them in the test/ folder of repos, e.g.:

- https://github.com/rasterio/rasterio/tree/1.3.11/tests/data
- https://github.com/cogeotiff/rio-tiler/tree/6.7.0/tests/fixtures
- https://github.com/locationtech/geotrellis/tree/v3.7.1/raster/data

> 3. One image using multiple ModelTiePoints (or ground control points)

Only dataset with GCPs I'm aware of are Sentinel-1 GRD files but they're quite big... Maybe see this comment for ideas - https://github.com/gjoseph92/stackstac/pull/182#issuecomment-1256574850, specifically the pointer to how rioxarray generates a synthetic GeoTIFF with GCPs.

Oh, and we might want to be careful with having too many GeoTIFF files commited to git history. If it's possible to generate synthetic images on-the-fly and test roundtrip write/read operations, that would be best. But I understand that we might still want a few test fixture files to test only the read-part.

**Comment by gschulze** [comment]
@kylebarron, @weiji14, @vincentsarago thank you all for your suggestions. I have to see whether there's something usable for that purpose; otherwise, I'll probably create a test image myself.
